lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/client/uploader/JobFolder.java

```diff
@@ -57,53 +57,57 @@
   }
 
   /**
-   * Uploads the given file to the DFS.
-   *
-   * @param localFile
+   * Upload given file to the DFS.
+   * @param localFile File on local FS to upload to the (remote) DFS.
    * @return the Path representing the file on the DFS.
-   * @throws IOException
+   * @throws IOException if unable to upload the file or local file cannot be read.
    */
   public Path upload(final File localFile) throws IOException {
+
     if (!localFile.exists()) {
       throw new FileNotFoundException(localFile.getAbsolutePath());
     }
 
     final Path source = new Path(localFile.getAbsolutePath());
     final Path destination = new Path(this.path, localFile.getName());
+
     try {
       this.fileSystem.copyFromLocalFile(source, destination);
-    } catch (final IOException e) {
-      LOG.log(Level.SEVERE, "Unable to upload {0} to {1}", new Object[]{source, destination});
-      throw e;
+    } catch (final IOException ex) {
+      LOG.log(Level.SEVERE, "Unable to upload " + source + " to " + destination, ex);
+      throw ex;
     }
-    LOG.log(Level.FINE, "Uploaded {0} to {1}", new Object[]{source, destination});
+
+    LOG.log(Level.FINE, "Uploaded {0} to {1}", new Object[] {source, destination});
```

Comment:
For the sake of consistency, can you please alter this log statement to use the syntax without `Object[]` ?